styleguide/derek/examples/_office365.html

```diff
@@ -2151,4 +2151,66 @@ <h4 class="center">The best part about all of our plans?<br/>
       </div>
     </div>
   </div>
-</div>
\ No newline at end of file
+
+  <!-- Enterprise Managed Services -->
+   <div class="container-fluid feature-container bg-crumpled-paper">
+
+      <div class="row">
```

Comment:
backgrounds need to be defined in a fluid container. there can be a container within that though. Just so the background spans the full width.